pkg/api/v1alpha1/vspheremachineconfig_types.go

```diff
@@ -18,6 +19,7 @@ type VSphereMachineConfigSpec struct {
 	StoragePolicyName string              `json:"storagePolicyName,omitempty"`
 	Template          string              `json:"template,omitempty"`
 	Users             []UserConfiguration `json:"users,omitempty"`
+	Taints            []corev1.Taint      `json:"taints,omitempty"`
```

Comment:
should define our own eksa taints struct instead of relying on upstream i think

Reply:
curious, what's the concern?

Reply:
so we have better control over what structure we define i think